docs/articles/kind-deploy.md

```diff
@@ -0,0 +1,220 @@
+# Using kind for Deployment Testing
+
+> :point_right: Use [kind](https://kind.sigs.k8s.io/) to try out zot deployment with Kubernetes.
+
+This article describes how to create a kind cluster that includes a local zot registry. 
+
+## Deploying the cluster and registry
+
+The procedure described installs a kind cluster with a zot registry at `localhost:5001` and then loads and runs a "hello" app to test the installation. Although the procedure is given as a series of steps, you can find [a complete shell script](#reference-a-complete-script) to perform these steps at the end of this article.
+
+> :pencil2: This article is based on [Create A Cluster And Registry](https://kind.sigs.k8s.io/docs/user/local-registry/#create-a-cluster-and-registry), which you can find on the [kind website](https://kind.sigs.k8s.io/).
+
+
+### Step 1: Prepare the environment 
+
+The following packages must be installed:
+
+- docker
+- kubernetes
+- kind
+- containerd
+- skopeo
+
+<div class="warning" style='padding:0.1em; background-color:#E9D8FD; color:#69337A'><span><p style='margin-top:1em; text-align:center'><b>Comments</b></p>
+<ol>
+<li><p style='margin-left:1em;'>
+Now that docker is being deprecated from K8s, should we be using equivalent commands from another package, such as podman or containerd?</p></li>
+<li><p style='margin-left:1em;'>Is this a correct list of required packages?
+</p></li></ol></div><br/>
+
+Execute the following shell commands to set environment variables.
+
+```shell
+set -o errexit
+
+# set no_proxy if applicable
+if [ ! -z "${no_proxy}" ]; then 
+  echo "Updating no_proxy environment variables";
+  export no_proxy=${no_proxy},kind-registry;
+  export NO_PROXY=${no_proxy};
+fi
+```
+
+### Step 2: Create a registry container
+
+Create a "kind-registry" container, pulling a zot binary from the GitHub Container Registry (ghcr.io).  
+
+> :pencil2: This example uses a minimal (no extensions) zot image for an AMD-based linux server. Other available images are described at [the zot releases page](https://github.com/project-zot/zot/releases) in GitHub.
+
+```shell
+{% raw %}
+# create registry container unless it already exists
+reg_name='kind-registry'
+reg_port='5001'
+if [ "$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || true)" != 'true' ]; then
+  docker run \
+    -d --restart=always -p "127.0.0.1:${reg_port}:5000" --name "${reg_name}" \
+    ghcr.io/project-zot/zot-minimal-linux-amd64:latest
```

Comment:
Add a note that `ghcr.io/project-zot/zot-minimal-linux-amd64:latest` with whatever zot version one wants

Reply:
Added suggestion in Step 2 to replace 'latest' with release number if desired.